sci-electronics/petrify/petrify-4.2-r1.ebuild

```
@@ -1,35 +1,32 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=8
 
 DESCRIPTION="Synthesize Petri nets into asynchronous circuits"
 HOMEPAGE="http://www.lsi.upc.edu/~jordicf/petrify/"
 SRC_URI="http://www.lsi.upc.edu/~jordicf/petrify/distrib/petrify-4.2-linux.tgz"
+S="${WORKDIR}"/${PN}
 
 LICENSE="Old-MIT"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE=""
 
 RDEPEND="media-gfx/graphviz"
-DEPEND=""
 
 RESTRICT="strip"
 
-QA_PREBUILT="/opt/petrify/petrify"
-
-S="${WORKDIR}"/${PN}
+QA_PREBUILT="opt/petrify/petrify"
 
 src_install() {
 	exeinto /opt/petrify
 	doexe bin/petrify lib/petrify.lib
 	dosym petrify /opt/petrify/draw_astg
 	dosym petrify /opt/petrify/write_sg
 
-	dodoc doc/*
+	dodoc -r doc/.
 	doman man/man1/*
 
 	dodir /etc/env.d
-	echo "PATH=${EPREFIX}/opt/petrify" > "${ED}"/etc/env.d/00petrify
+	newenvd - 00petrify <<<"PATH=${EPREFIX}/opt/petrify"
```

```suggestion
	newenvd - 00petrify <<- _EOF_
		PATH="${EPREFIX}/opt/petrify"
	_EOF_
```
which is a bit more readable (and keeps the quotes in the file)

---

Ok, this looks nicer